### PR TITLE
Remove nerd term "use case", fix typos

### PR DESCRIPTION
**Pull request by dngray**
- [x] Please check this box to confirm you have disclosed any relevant conflicts of interest in your post.
- [x] Please check this box to confirm your agreement to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute your contribution as part of our project.
- [x] Please check this box to confirm you are the sole author of this work, or that any additional authors will also reply to this PR on GitHub confirming their agreement to these terms.

### CODE REVIEW

File: docs/os/qubes-overview.en.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Each Qubes application has a [colored border](https://www.qubes-os.org/screensho
 
 ## Why Should I use Qubes?
 
-Qubes OS is useful if your [threat model](../basics/threat-modeling.md) requires strong compartmentalization and security, such as if you think you'll be opening untrusted files from untrusted sources. A typical use-case for Qubes OS is for opening documents from an unknown source.
+Qubes OS is useful if your [threat model](../basics/threat-modeling.md) requires strong compartmentalization and security, such as if you think you'll be opening untrusted files from untrusted sources. A typical reason for using Qubes OS is to open documents from unknown sources.
```
Comment (jonaharagon) [inline docs/os/qubes-overview.en.md:21]: do these two sentences just say the same thing? lol

Comment (dngray) [reply docs/os/qubes-overview.en.md:21]: s/to open/when opening?

**Comment by dngray** [comment]
Last two to resolve:

- https://crowdin.com/translate/privacyguides/448/en-hu?filter=basic&value=0#q=39992
- https://crowdin.com/translate/privacyguides/448/en-hu?filter=basic&value=0#q=39998

File: docs/os/qubes-overview.en.md (hunk 1 above)
Comment (sharp-tailed) [reply docs/os/qubes-overview.en.md:21]: > do these two sentences just say the same thing? lol

Actually, yes, they are pretty much the same thing. I can't believe this slipped past my final edits, too.

**Pull request merged**